pkg/kubernetes/node/kubernetes.go

```diff
@@ -149,3 +149,23 @@ func (k *Kubeclient) List(opts metav1.ListOptions) (*corev1.NodeList, error) {
 
 	return k.list(cli, opts)
 }
+
+// NumberOfNodes returns the number of nodes registered in a Kubernetes cluster
+func NumberOfNodes() (int, error) {
+	nodes, err := NewKubeClient().List(metav1.ListOptions{})
```

Comment:
do we already have the cluster role for listing nodes in cstor-csi?

Reply:
Yes its there